django_netjsongraph/migrations/0002_topology_published.py

```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.9 on 2015-12-26 17:05
-from __future__ import unicode_literals
```

Comment:
@nemesisdesign can you check this change.
Looks good to me and I tested, everything is working with this change but because it's a change in migration that's released just wanted you to see this and confirm if it's fine. :smile: